docs/LV_client/tools/mono_plot_radar_target_by_name.md

```diff
@@ -0,0 +1,33 @@
+# mono_plot_radar_target_by_name.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_plot_radar_target_by_name.png"/>
+</p>
+
+### Description
+
+Tool used to generate plot to compare AoA, Range and Velocity between the GT targets and the target list from the Radar sensor. This tool only tracks one target by the specified target on the **Name of Target** control 
+
+### Inputs
+
+- **Radar Sample:**  Refer to the **radarSample.ctl** description 
+
+- **Name of target:**  Name of the target to track and generated plots. This target
+name will be used to generate the AoA, Range and Velocity
+plots comparing the Ground Truth information and the
+information returned by the sensor
+ 
```

Comment:
Target name of tracking and generating plots. This target
name will be used to generate the AoA, Range and Velocity
plots that compare the Ground Truth information and the
information returned by the sensor